### PR TITLE
proper error when computing poynting vector of jax-traced field data

### LINKED ISSUE #1531: JaxFieldData bug in 'get_poynting_vector' method
**Describe the bug**
The 'get_poynting_vector' method of JaxSimulationData is throwing an error. It appears it's treating the JaxSimulationData the same way as tidy3d SimulationData.

**To Reproduce**
Simply use the 'get_poynting_vector' method on any tda.JaxSimulationData object. For example see the first screenshot.

**Screenshots**
When running
image
I'm getting the error
image

### PR DESCRIPTION
**Pull request by tylerflex**
fixes #1531

### CODE REVIEW

File: tidy3d/plugins/adjoint/components/data/sim_data.py [hunk 1]
```diff
@@ -54,6 +54,37 @@ class JaxSimulationData(SimulationData, JaxObject):
...
+            DataArray containing the Poynting vector calculated based on the field components
+            colocated at the center locations of the Yee grid.
+        """
+
+        if field_monitor_name in self.output_monitor_data:
+            raise NotImplementedError(
+                "Adjoint support for differentiation with respect to Poynting vector not available."
+                " If you would like to request this feature for future versions, "
+                "please make an issue on the tidy3d front end repository "
+                "'https://github.com/flexcompute/tidy3d/issues'."
```
Comment (momchil-flex) [inline tidy3d/plugins/adjoint/components/data/sim_data.py]: How about we make the issue ourselves and we just say here that it is not yet available?

Comment (tylerflex) [reply tidy3d/plugins/adjoint/components/data/sim_data.py]: ok, just removed the last three lines in this message

Comment (tylerflex) [reply tidy3d/plugins/adjoint/components/data/sim_data.py]: and made a new issue for this.

**Comment by tylerflex** [comment]
@momchil-flex while you're merging stuff, I just rebased this for 2.7 so should be all set.

**Comment by momchil-flex** [comment]
Great, thanks, I wanted to merge it yeah but I was too lazy to fix myself. :D

**Pull request merged**